TTMouseclickSimulator/SampleProjects/ToontownRewritten/Auto Fishing - Estate.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SimulatorProject xmlns="https://github.com/TTExtensions/MouseClickSimulator">
+  <Title>Automatic Fishing in Toon Estate</Title>
+  <Description>
+The Toon will automatically fish in your estate.
+Before you click on Start, make sure that
+• your fish bucket is empty,
+• you have enough JellyBeans for 20 casts,
+• your toon is standing on the board nearest to the bucket
+Note: Doodles may cause the simulator to fail.
+  </Description>
+
+  <MainAction>
+
+    <Compound type="Sequential">
+
+      <!-- 
+	  		Use the automatic fishing action for estate 20 times,
+       		and ensure Toon remains fishing instead of going to bed.
+      -->
+      <Loop count="5"> 
```

Comment:
Is there a reason for only repeating the auto fishing 5 times, then pressing the down/up keys, and then fishing again? If the toon has enough jellybeans for 20 casts, I think this should probably not be needed, and it should be possible to do 19 or 20 casts in a row.

Reply:
Probably not, I don't even remember why I did this but it may have had something to do with preventing timeout? If it works without though then that change would make sense.